boggs/src/main/java/com/hartwig/healthchecks/boggs/healthcheck/mapping/MappingExtractor.java

```diff
@@ -17,65 +8,73 @@
 import com.hartwig.healthchecks.boggs.model.report.MappingReport;
 import com.hartwig.healthchecks.common.exception.EmptyFileException;
 import com.hartwig.healthchecks.common.util.CheckType;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
 
 public class MappingExtractor extends BoggsExtractor {
-	private static final String EMPTY_FILES_ERROR = "Found empty Summary files and/or fastqc_data under path -> %s";
+    private static final String EMPTY_FILES_ERROR = "Found empty Summary files and/or fastqc_data under path -> %s";
 
-	private static final String REALIGN = "realign";
-	private static final String SAMPLE_PREFIX = "CPCT";
-	private static final String REF_SAMPLE_SUFFIX = "R";
-	private static final String FLAGSTAT_SUFFIX = ".flagstat";
+    private static final String REALIGN = "realign";
+    private static final String SAMPLE_PREFIX = "CPCT";
+    private static final String REF_SAMPLE_SUFFIX = "R";
+    private static final String FLAGSTAT_SUFFIX = ".flagstat";
 
-	@NotNull
-	private final FlagStatParser flagstatParser;
+    @NotNull
+    private final FlagStatParser flagstatParser;
 
-	public MappingExtractor(@NotNull FlagStatParser flagstatParser) {
-		this.flagstatParser = flagstatParser;
-	}
+    public  MappingExtractor(@NotNull final FlagStatParser flagstatParser) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It is a good practice to call super() in a constructor](https://www.codacy.com/app/hmf/health-checks/file/2780941525/issues/source?bid=3363458&fileBranchId=3365457#l31)